purchase_request_to_requisition/models/purchase_request.py

```diff
@@ -17,24 +17,27 @@
 class PurchaseRequestLine(models.Model):
     _inherit = "purchase.request.line"
 
-    @api.one
+    @api.multi
     @api.depends('requisition_lines')
-    def _get_is_editable(self):
-        super(PurchaseRequestLine, self)._get_is_editable()
+    def _compute_is_editable(self):
+        self.ensure_one()
```

Comment:
@api.multi should have a loop on self and remove self.ensure_one() it will give a stack trace if the field is shown into a tree view